src/main/resources/templates/signup2.html

```diff
@@ -199,6 +222,8 @@ <h3 th:text="#{signup.applyCreateTeam.header}">Apply to create a new team</h3>
 			                            <li>&nbsp;</li>
 			                            <li class="active">&nbsp;</li>
 			                        </ul>
+
+									<p class="account-change-fail" th:if="${#fields.hasErrors('joinTeamName')}"><i class="fa fa-exclamation-circle"></i> <span th:errors="*{joinTeamName}"></span></p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)